### PR TITLE
Dutch Translation 

### PR DESCRIPTION
**Pull request by SweptWasTaken**
Dutch Translation for the Rimworld Mod.

### CODE REVIEW

File: Languages/Dutch [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  <MpWrongMultiplayerVersionInfo>De server wordt gehost op Multiplayer {0} (protocol {1}).</MpWrongMultiplayerVersionInfo>
+  
+  <MpWrongDefs>Modeldefinitie mismatch</MpWrongDefs>
+  <MpWrongDefsInfo>Zorg ervoor dat de door u geïnstalleerde wijzigingen, de versies en de bestelling overeenkomen met die van de host.</MpWrongDefsInfo>
+  <MpModList>Wijzigingslijst</MpModList>
+  <MpSeeModList>Zie wijzigingslijst</MpSeeModList>
+  
+  <MpServerClosed>Server gesloten</MpServerClosed>
+  <MpServerFull>De server is vol</MpServerFull>
+  <MpKicked>Je bent geschopt...</MpKicked>
```
Comment (MilanDierick) [inline Languages/Dutch]: This looks a bit iffy... Using Google Translate?

File: Languages/Dutch [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  <MpInvalidUsernameLength>Gebruiker heeft ongeldige lengte</MpInvalidUsernameLength>
+  <MpChangeUsernameInfo>U kunt uw gebruiker wijzigen in de wijzigingsinstellingen.</MpChangeUsernameInfo>
+  
+  <MpPlayerConnected>{0} heeft zich aangesloten</MpPlayerConnected>
+  <MpPlayerDisconnected>{0} is vertrokken</MpPlayerDisconnected>
+  
+  <MpReplayOutdated>(Outdated)</MpReplayOutdated>
+  <MpReplayOutdatedDesc1>Deze herhaling is gemaakt voor de protocolversie {0}, maar je bent bezig met de versie die je gebruikt... {1}.</MpReplayOutdatedDesc1>
+  <MpReplayOutdatedDesc2>De herhalingen worden mogelijk niet correct afgespeeld.</MpReplayOutdatedDesc2>
+  <MpReplayOutdatedDesc3>Mogelijke onverenigbaarheden worden vermeld met de releaseverleden.</MpReplayOutdatedDesc3>
```
Comment (MilanDierick) [inline Languages/Dutch]: I'm sorry, what?

File: Languages/Dutch [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  <MpReplayOutdatedDesc4>Dit bestand bevat aanvullende informatie en de hosting ervan zou goed moeten werken.</MpReplayOutdatedDesc4>
+  
+  <MpServerCloseConfirmation>Weet u zeker dat u de server wilt sluiten? Onbewaarde vooruitgang zal verloren gaan.</MpServerCloseConfirmation>
+  
+  <MpHostServer>Host een server</MpHostServer>
+  <MpSaveReplay>Opslaan van herhaling</MpSaveReplay>
+  
+  <MpConvert>Omzetten naar singleplayer</MpConvert>
+  <MpConverting>Het omzetten</MpConverting>
```
Comment (MilanDierick) [inline Languages/Dutch]: Sounds iffy, would need to check the English translation...

File: Languages/Dutch [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  
+  <MpUsername>Gebruikersnaam</MpUsername>
+  <MpShowPlayerCursors>Toon speler cursors</MpShowPlayerCursors>
+  <MpAutoAcceptSteam>Auto-accepteren stoom</MpAutoAcceptSteam>
```
Comment (MilanDierick) [inline Languages/Dutch]: This is translated with Google Translate...

File: Languages/Dutch [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+  <MpAutoAcceptSteamDesc>Accepteer automatisch alle inkomende aanvragen voor een stoomverbinding.</MpAutoAcceptSteamDesc>
```
Comment (MilanDierick) [inline Languages/Dutch]: Again, Google Translate...

File: Languages/Dutch [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+  <MpTransparentChat>Transparant gesprek</MpTransparentChat>
+  
+  <MpAutosaveSlots>Autosave groeven</MpAutosaveSlots>
```
Comment (MilanDierick) [inline Languages/Dutch]: Again, Google Translate...

File: Languages/Dutch [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+  <MpAutosaveSlotsDesc>Hoeveel autosave-bestanden te bewaren.</MpAutosaveSlotsDesc>
+  
+  <MpTraderLeavesIn>Blijft in {0}</MpTraderLeavesIn>
+  
+  <MpChatButton>Chat</MpChatButton>
+  <MpTradingButton>Handel</MpTradingButton>
+  
+  <MpChatPlayers>Spelers ({0}):</MpChatPlayers>
+  <MpSend>Verzenden</MpSend>
+  <MpSteamAcceptTitle>Accepteer stoomspelers</MpSteamAcceptTitle>
```
Comment (MilanDierick) [inline Languages/Dutch]: Again, Google Translate...

File: Languages/Dutch [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+  <MpSteamAccepted>Speler is geaccepteerd</MpSteamAccepted>
+  
+  <MpConnectingTo>Verbinding maken met {0}:{1}</MpConnectingTo>
+  <MpConnected>Aangesloten.</MpConnected>
+  <MpDownloading>Downloaden ({0}%)</MpDownloading>
+  
+  <MpSteamConnectingTo>Verbinding maken met een spel dat wordt gehost door {0}</MpSteamConnectingTo>
+  <MpSteamConnectingWaiting>Wachten tot de gastheer het accepteert...</MpSteamConnectingWaiting>
+  
+  <MpTicksBehind>U bent {0} teekken achter.</MpTicksBehind>
```
Comment (MilanDierick) [inline Languages/Dutch]: Again, Google Translate...

File: Languages/Dutch [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+  <MpLowerGameSpeed>Overweeg het verlagen van de spelsnelheid.</MpLowerGameSpeed>
+  <MpForceCatchUp>Klik om te dwingen de achterstand in te halen.</MpForceCatchUp>
+  
+  <MpDesynced>De staat van het spel heeft het spel ontzenuwd.</MpDesynced>
```
Comment (MilanDierick) [inline Languages/Dutch]: Again, Google Translate...

File: Languages/Dutch [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+  <MpTryResync>Probeer resync</MpTryResync>
+  
+  <MpSaveReplayAs>Sla replay op als</MpSaveReplayAs>
+  <MpWillOverwrite>Zal overschrijven</MpWillOverwrite>
+  <MpReplaySaved>Replay opgeslagen</MpReplaySaved>
+  
+  <MpGameName>Naam van het spel</MpGameName>
+  <MpMaxPlayers>Max spelers</MpMaxPlayers>
+  <MpAutosaveEvery>Sla elke</MpAutosaveEvery>
```
Comment (MilanDierick) [inline Languages/Dutch]: Iffy...

File: Languages/Dutch [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+  <MpAutosaveMinutes>notulen</MpAutosaveMinutes>
```
Comment (MilanDierick) [inline Languages/Dutch]: I'm sorry, what?

File: Languages/Dutch [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+  <MpLanDesc1>Zend het spel uit naar uw lokale netwerk.</MpLanDesc1>
+  <MpLanDesc2>Opgelost LAN-adres: {0}</MpLanDesc2>
+  <MpArbiterDesc>Een spelvoorbeeld dat op de achtergrond loopt en helpt bij het oplossen van desync.</MpArbiterDesc>
```
Comment (MilanDierick) [inline Languages/Dutch]: "Spelvoorbeeld" ?

File: Languages/Dutch [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+  <MpAsyncTimeDesc>Afzonderlijke tijdcontroles voor elke kaart en de planeet.</MpAsyncTimeDesc>
+  <MpExperimentalFeature>Experimentele functie</MpExperimentalFeature>
+  
+  <MpHostButton>Gastheer</MpHostButton>
+  <MpWatchReplay>Horloge</MpWatchReplay>
```
Comment (MilanDierick) [inline Languages/Dutch]: Again, Google Translate...

File: Languages/Dutch [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+  <MpNothingSelected>Niets geselecteerd</MpNothingSelected>
+  <MpNoSaves>Multiplayer kan alleen worden afgespeeld vanuit reeds bestaande saves</MpNoSaves>
+  
+  <MpHostIngame>Verhuurder van dit spel</MpHostIngame>
```
Comment (MilanDierick) [inline Languages/Dutch]: Again, Google Translate...

File: Languages/Dutch [hunk 15]
```diff
@@ -0,0 +1,128 @@
...
+  <MpHostReplay>Verhuurder van een replay</MpHostReplay>
```
Comment (MilanDierick) [inline Languages/Dutch]: Again, Google Translate...

File: Languages/Dutch [hunk 16]
```diff
@@ -0,0 +1,128 @@
...
+  <MpHostSavefile>Verhuurders van een spaarboekje</MpHostSavefile>
```
Comment (MilanDierick) [inline Languages/Dutch]: What?

File: Languages/Dutch [hunk 17]
```diff
@@ -0,0 +1,128 @@
...
+  
+  <MpJoinButton>Word lid</MpJoinButton>
+  <MpConnectButton>Verbinden</MpConnectButton>
+  <MpLanSearching>Zoeken</MpLanSearching>
+  
+  <MpNotConnectedToSteam>Niet aangesloten op stoom</MpNotConnectedToSteam>
```
Comment (MilanDierick) [inline Languages/Dutch]: Again, Google Translate...

File: Languages/Dutch [hunk 18]
```diff
@@ -0,0 +1,128 @@
...
+  <MpNoFriendsPlaying>Geen vrienden die momenteel RimWorld spelen</MpNoFriendsPlaying>
+  <MpPlayingRimWorld>Spelen RimWorld spelen</MpPlayingRimWorld>
```
Comment (MilanDierick) [inline Languages/Dutch]: That sentence doesn't make any sense...

File: Languages/Dutch [hunk 19]
```diff
@@ -0,0 +1,128 @@
...
+  <MpNotInMultiplayer>Niet in multiplayer</MpNotInMultiplayer>
+  
+  <MpPauseOnAutosave>Pauze op Autosave</MpPauseOnAutosave>
+  <MpLan>LAN</MpLan>
+  <MpDirect>Direct</MpDirect>
+  <MpSteam>Stoom</MpSteam>
```
Comment (MilanDierick) [inline Languages/Dutch]: Again, Google Translate...

File: Languages/Dutch [hunk 20]
```diff
@@ -0,0 +1,128 @@
...
+  <MpHostTab>Gastheer</MpHostTab>
```
Comment (MilanDierick) [inline Languages/Dutch]: Again, Google Translate...

File: Languages/Dutch [hunk 21]
```diff
@@ -0,0 +1,128 @@
...
+  
+  <MpMultiplayer>Multiplayer</MpMultiplayer>
+  <MpSingleplayer>Singleplayer</MpSingleplayer>
+  
+  <MpNotAvailable>Niet beschikbaar in de meerspelerstand.</MpNotAvailable>
+  <MpCallingFactionNotAvailable>Andere facties bellen is momenteel niet beschikbaar in de multiplayer.</MpCallingFactionNotAvailable>
```
Comment (MilanDierick) [inline Languages/Dutch]: Again, Google Translate...

**Review by MilanDierick — DISMISSED** [review]
I have indicated the most prominent problems. You clearly used Google Translate to get the job done. This does not meet our quality standards. Please rewrite the translation without Google Translate, or we will close the pull request.

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 22]
```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<LanguageData>
+
+  <MpSimulating>Simuleren</MpSimulating>
+  <MpSimulatingServer>Simuleren van de server</MpSimulatingServer>
+  <MpLoading>Laden</MpLoading>
+  <MpSaving>Besparing</MpSaving>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "Aan het opslaan"
"Bezig met oplsaan"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 23]
```diff
@@ -0,0 +1,128 @@
...
+  
+  <MpInvalidPort>Ongeldige poort</MpInvalidPort>
+  <MpInvalidAddress>Ongeldig adres</MpInvalidAddress>
+  <MpInvalidGameName>Ongeldige spelnaam</MpInvalidGameName>
+  
+  <MpWrongProtocol>Verkeerde protocolversie</MpWrongProtocol>
+  <MpWrongMultiplayerVersionInfo>De server wordt gehost op Multiplayer {0} (protocol {1}).</MpWrongMultiplayerVersionInfo>
+  
+  <MpWrongDefs>Modeldefinitie mismatch</MpWrongDefs>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "Modeldefinitie komt niet overeen"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 24]
```diff
@@ -0,0 +1,128 @@
...
+  <MpWrongDefsInfo>Zorg ervoor dat de door u geïnstalleerde wijzigingen, de versies en de bestelling overeenkomen met die van de host.</MpWrongDefsInfo>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "bestelling"?

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 25]
```diff
@@ -0,0 +1,128 @@
...
+  <MpModList>Wijzigingslijst</MpModList>
+  <MpSeeModList>Zie wijzigingslijst</MpSeeModList>
+  
+  <MpServerClosed>Server gesloten</MpServerClosed>
+  <MpServerFull>De server is vol</MpServerFull>
+  <MpKicked>Je bent geschopt...</MpKicked>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: Still looks a bit iffy...

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 26]
```diff
@@ -0,0 +1,128 @@
...
+
+  <MpInvalidUsernameAlreadyPlaying>Gebruiker al online</MpInvalidUsernameAlreadyPlaying>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "Gebruiker is al aanwezig in de game"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 27]
```diff
@@ -0,0 +1,128 @@
...
+  <MpInvalidUsernameChars>Gebruiker heeft ongeldige tekens</MpInvalidUsernameChars>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "Gebruikersnaam bevat ongeldige tekens"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 28]
```diff
@@ -0,0 +1,128 @@
...
+  <MpInvalidUsernameLength>Gebruiker heeft ongeldige lengte</MpInvalidUsernameLength>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "Gebruikersnaam heeft een ongeldige lengte"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 29]
```diff
@@ -0,0 +1,128 @@
...
+  <MpChangeUsernameInfo>U kunt uw gebruiker wijzigen in de wijzigingsinstellingen.</MpChangeUsernameInfo>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "U kunt uw gebruikersnaam wijzigen in de wijzigingsinstellingen."

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 30]
```diff
@@ -0,0 +1,128 @@
...
+  
+  <MpPlayerConnected>{0} heeft zich aangesloten</MpPlayerConnected>
+  <MpPlayerDisconnected>{0} is vertrokken</MpPlayerDisconnected>
+  
+  <MpReplayOutdated>(Outdated)</MpReplayOutdated>
+  <MpReplayOutdatedDesc1>Deze herhaling is gemaakt voor de protocolversie {0}, maar je bent bezig met de versie die je gebruikt... {1}.</MpReplayOutdatedDesc1>
+  <MpReplayOutdatedDesc2>De herhalingen worden mogelijk niet correct afgespeeld.</MpReplayOutdatedDesc2>
+  <MpReplayOutdatedDesc3>Mogelijke onverenigbaarheden worden vermeld met de releaseverleden.</MpReplayOutdatedDesc3>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: I'm sorry, what?

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 31]
```diff
@@ -0,0 +1,128 @@
...
+  <MpReplayOutdatedDesc4>Dit bestand bevat aanvullende informatie en de hosting ervan zou goed moeten werken.</MpReplayOutdatedDesc4>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml:37]: I'm sorry, what?

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 32]
```diff
@@ -0,0 +1,128 @@
...
+  
+  <MpServerCloseConfirmation>Weet u zeker dat u de server wilt sluiten? Onbewaarde vooruitgang zal verloren gaan.</MpServerCloseConfirmation>
+  
+  <MpHostServer>Host een server</MpHostServer>
+  <MpSaveReplay>Opslaan van herhaling</MpSaveReplay>
+  
+  <MpConvert>Omzetten naar singleplayer</MpConvert>
+  <MpConverting>Het omzetten</MpConverting>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: Google Translate...

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 33]
```diff
@@ -0,0 +1,128 @@
...
+  
+  <MpUsername>Gebruikersnaam</MpUsername>
+  <MpShowPlayerCursors>Toon speler cursors</MpShowPlayerCursors>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "Toon cursors van spelers"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 34]
```diff
@@ -0,0 +1,128 @@
...
+  <MpAutoAcceptSteam>Auto-accepteren stoom</MpAutoAcceptSteam>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: Steam

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 35]
```diff
@@ -0,0 +1,128 @@
...
+  <MpAutoAcceptSteamDesc>Accepteer automatisch alle inkomende aanvragen voor een stoomverbinding.</MpAutoAcceptSteamDesc>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: Steam

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 36]
```diff
@@ -0,0 +1,128 @@
...
+  <MpTransparentChat>Transparant gesprek</MpTransparentChat>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "Transparante chat"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 37]
```diff
@@ -0,0 +1,128 @@
...
+  
+  <MpAutosaveSlots>Autosave groeven</MpAutosaveSlots>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: Google Translate...

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 38]
```diff
@@ -0,0 +1,128 @@
...
+  <MpAutosaveSlotsDesc>Hoeveel autosave-bestanden te bewaren.</MpAutosaveSlotsDesc>
+  
+  <MpTraderLeavesIn>Blijft in {0}</MpTraderLeavesIn>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: Sounds iffy: "TraderLeavesIn" -> "Handelaar vertrekt in ..."

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 39]
```diff
@@ -0,0 +1,128 @@
...
+  <MpTradingButton>Handel</MpTradingButton>
+  
+  <MpChatPlayers>Spelers ({0}):</MpChatPlayers>
+  <MpSend>Verzenden</MpSend>
+  <MpSteamAcceptTitle>Steam-spelers accepteren</MpSteamAcceptTitle>
+  <MpSteamAcceptDesc>Klik om te accepteren</MpSteamAcceptDesc>
+  <MpSteamAccepted>Speler is geaccepteerd</MpSteamAccepted>
+  
+  <MpConnectingTo>Verbinding maken met {0}:{1}</MpConnectingTo>
+  <MpConnected>Aangesloten.</MpConnected>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml:68]: "Verbonden"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 40]
```diff
@@ -0,0 +1,128 @@
...
+  <MpDownloading>Downloaden ({0}%)</MpDownloading>
+  
+  <MpSteamConnectingTo>Een verbinding met {0} aan het maken</MpSteamConnectingTo>
+  <MpSteamConnectingWaiting>We wachten op een acceptatie van de host</MpSteamConnectingWaiting>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "We wachten op een bevestiging van de host"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 41]
```diff
@@ -0,0 +1,128 @@
...
+  
+  <MpTicksBehind>U bent {0} ticks vroeger.</MpTicksBehind>
+  <MpLowerGameSpeed>Overweeg het verlagen van de spelsnelheid.</MpLowerGameSpeed>
+  <MpForceCatchUp>Klik om te dwingen de achterstand in te halen.</MpForceCatchUp>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: Waar is het onderwerp in deze zin?

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 42]
```diff
@@ -0,0 +1,128 @@
...
+  
+  <MpDesynced>De staat van het spel heeft het spel ontzenuwd.</MpDesynced>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: I'm sorry, what?

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 43]
```diff
@@ -0,0 +1,128 @@
...
+  <MpTryResync>Probeer resync</MpTryResync>
+  
+  <MpSaveReplayAs>Sla replay op als</MpSaveReplayAs>
+  <MpWillOverwrite>Zal overschrijven</MpWillOverwrite>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: Please look at the English translation file in order to translate this

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 44]
```diff
@@ -0,0 +1,128 @@
...
+  <MpReplaySaved>Replay opgeslagen</MpReplaySaved>
+  
+  <MpGameName>Naam van dit spel</MpGameName>
+  <MpMaxPlayers>Limiet van spelers</MpMaxPlayers>
+  <MpAutosaveEvery>Automatisch elke</MpAutosaveEvery>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: Please look at the English translation file in order to translate this

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 45]
```diff
@@ -0,0 +1,128 @@
...
+  <MpAutosaveMinutes>Minuten</MpAutosaveMinutes>
+  <MpLanDesc1>Zend dit speel uit op uw lokale netwerk</MpLanDesc1>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "Host dit spel uit op uw lokaal netwerk"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 46]
```diff
@@ -0,0 +1,128 @@
...
+  <MpLanDesc2>Lokaal netwerkadres gevonden: {0}</MpLanDesc2>
+  <MpArbiterDesc>Een instantie die draait op de achtergrond en helpt met de voorlichting van desync</MpArbiterDesc>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "Een proces dat draait op de achtergrond en helpt met de detectie van desyncs

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 47]
```diff
@@ -0,0 +1,128 @@
...
+  <MpHostButton>Hosten</MpHostButton>
+  <MpWatchReplay>Kijken</MpWatchReplay>
+  <MpNothingSelected>Niets geselected</MpNothingSelected>
+  <MpNoSaves>Multiplayermodus kan alleen gebruikt worden bij spellen die al bestaan</MpNoSaves>
+  
+  <MpHostIngame>Uit dit spel hosten</MpHostIngame>
+  <MpHostReplay>Uit een herhaling hosten</MpHostReplay>
+  <MpHostSavefile>Host een savefile</MpHostSavefile>
+  
+  <MpJoinButton>Word lid</MpJoinButton>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: Google Translate...

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 48]
```diff
@@ -0,0 +1,128 @@
...
+  <MpConnectButton>Verbinden</MpConnectButton>
+  <MpLanSearching>Aan het opzoeken</MpLanSearching>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "Aan het zoeken"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 49]
```diff
@@ -0,0 +1,128 @@
...
+  <MpLan>LAN</MpLan>
+  <MpDirect>Direct</MpDirect>
+  <MpSteam>Steam</MpSteam>
+  <MpHostTab>Hosten</MpHostTab>
+  
+  <MpMultiplayer>Multiplayermodus</MpMultiplayer>
+  <MpSingleplayer>Singleplayermodus</MpSingleplayer>
+  
+  <MpNotAvailable>Niet beschikbaar in de meerspelerstand.</MpNotAvailable>
+  <MpCallingFactionNotAvailable>Andere facties bellen is momenteel niet beschikbaar in de multiplayer.</MpCallingFactionNotAvailable>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "Andere facties contacteren is momenteel niet mogelijk in Multiplayer.

**Review by MilanDierick — CHANGES_REQUESTED** [review]
Most changes I requested last time still haven't been resolved properly. I also added a bunch more requested changes.

**Comment by MilanDierick** [comment]
@SweptWasTaken Any updates on the remaining change requests?

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 50]
```diff
@@ -0,0 +1,130 @@
...
+
+  <MpInvalidUsernameAlreadyPlaying>Gebruiker is al aanwezig in de game</MpInvalidUsernameAlreadyPlaying>
+  <MpInvalidUsernameChars>Gebruikersnaam bevat ongeldige tekens</MpInvalidUsernameChars>
+  <MpInvalidUsernameLength>Gebruikersnaam heeft een ongeldige lengte</MpInvalidUsernameLength>
+  <MpChangeUsernameInfo>U kunt uw gebruikersnaam wijzigen in de wijzigingsinstellingen.</MpChangeUsernameInfo>
+  
+  <MpPlayerConnected>{0} heeft zich aangesloten</MpPlayerConnected>
+  <MpPlayerDisconnected>{0} is vertrokken</MpPlayerDisconnected>
+  
+  <MpReplayOutdated>(Outdated)</MpReplayOutdated>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "Niet up-to-date"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 51]
```diff
@@ -0,0 +1,130 @@
...
+  <MpReplayOutdatedDesc1>Deze herhaling is gemaakt voor de protocolversie {0}, maar je bent bezig met de versie die je gebruikt... {1}.</MpReplayOutdatedDesc1>
+  <MpReplayOutdatedDesc2>De herhalingen worden mogelijk niet correct afgespeeld.</MpReplayOutdatedDesc2>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "mogelijks"

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 52]
```diff
@@ -0,0 +1,130 @@
...
+  <MpReplayOutdatedDesc3>Mogelijke incompatibiliteiten worden vermeld in het releaseverleden.</MpReplayOutdatedDesc3>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml:36]: What is the English translation for this?

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 53]
```diff
@@ -0,0 +1,130 @@
...
+  <MpReplayOutdatedDesc4>Dit bestand bevat aanvullende informatie en de hosting ervan zou goed moeten werken.</MpReplayOutdatedDesc4>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml:37]: What is the English translation for this?

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 54]
```diff
@@ -0,0 +1,130 @@
...
+  <MpHostServer>Host een server</MpHostServer>
+  <MpSaveReplay>Opslaan van herhaling</MpSaveReplay>
+  
+  <MpConvert>Omzetten naar singleplayer</MpConvert>
+  <MpConverting>Aan het omzetten</MpConverting>
+  
+  <MpUsername>Gebruikersnaam</MpUsername>
+  <MpShowPlayerCursors>Toon cursors van spelers</MpShowPlayerCursors>
+  <MpAutoAcceptSteam>Auto-accepteren Steam</MpAutoAcceptSteam>
+  <MpAutoAcceptSteamDesc>Accepteer automatisch alle inkomende aanvragen voor een stoomverbinding.</MpAutoAcceptSteamDesc>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: Steam!

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 55]
```diff
@@ -0,0 +1,130 @@
...
+  
+  <MpTicksBehind>U bent {0} ticks vroeger.</MpTicksBehind>
+  <MpLowerGameSpeed>Overweeg de spelsnelheid te verlagen.</MpLowerGameSpeed>
+  <MpForceCatchUp>Klik om achterstand inhalen te forceren.</MpForceCatchUp>
+  
+  <MpDesynced>De staat van het spel is niet (meer) gesynchroniseerd.</MpDesynced>
+  <MpTryResync>Probeer hersynchronisatie</MpTryResync>
+  
+  <MpSaveReplayAs>Sla replay op als</MpSaveReplayAs>
+  <MpWillOverwrite>Dit zal overschrijven</MpWillOverwrite>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: This is iffy, what is the English translation?

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 56]
```diff
@@ -0,0 +1,130 @@
...
+  <MpExperimentalFeature>Experimentele eigenschappen</MpExperimentalFeature>
+  
+  <MpHostButton>Hosten</MpHostButton>
+  <MpWatchReplay>Kijken</MpWatchReplay>
+  <MpNothingSelected>Niets geselecteerd</MpNothingSelected>
+  <MpNoSaves>Multiplayermodus kan alleen gebruikt worden bij spellen die al bestaan</MpNoSaves>
+  
+  <MpHostIngame>Uit dit spel hosten</MpHostIngame>
+  <MpHostReplay>Uit een herhaling hosten</MpHostReplay>
+  <MpHostSavefile>host vanuit bestaande savefile</MpHostSavefile>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: Capitalization

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 57]
```diff
@@ -0,0 +1,130 @@
...
+  <MpHostTab>Hosten</MpHostTab>
+  
+  <MpMultiplayer>Multiplayermodus</MpMultiplayer>
+  <MpSingleplayer>Singleplayermodus</MpSingleplayer>
+  
+  <MpNotAvailable>Niet beschikbaar in de meerspelerstand.</MpNotAvailable>
+  <MpCallingFactionNotAvailable>Andere facties contacteren is momenteel niet mogelijk in Multiplayer.</MpCallingFactionNotAvailable>
+  
+  <MpAggressiveTicking>Agressief tikken</MpAggressiveTicking>
+  <MpAggressiveTickingDesc>Probeer altijd automatisch de server in te halen op kosten van de FPS.</MpAggressiveTickingDesc>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml]: "...op kosten van FPS"

**Review by MilanDierick — CHANGES_REQUESTED** [review]

File: Languages/Dutch/Keyed/Multiplayer.xml (hunk 52 above)
Comment (SweptWasTaken) [reply Languages/Dutch/Keyed/Multiplayer.xml:36]: Possible incompatibilities are listed *in* the release history.

File: Languages/Dutch/Keyed/Multiplayer.xml (hunk 53 above)
Comment (SweptWasTaken) [reply Languages/Dutch/Keyed/Multiplayer.xml:37]: This file contains additional information and hosting from it *should work fine.*

File: Languages/Dutch/Keyed/Multiplayer.xml (hunk 55 above)
Comment (SweptWasTaken) [reply Languages/Dutch/Keyed/Multiplayer.xml]: This will overwrite "..."/Dit zal "..." overschrijven

Comment (MilanDierick) [reply Languages/Dutch/Keyed/Multiplayer.xml]: "Dit zal overschreven worden: "

**Review by MilanDierick — APPROVED** [review]

**Review by tindolt-zz — APPROVED** [review]
Looks good, also good idea on the comment at the end.

**Pull request merged**